src/Components/StatusPills/ColorPills.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { IColorPillsProps } from './StatusPills.types';
+import { getPillStyles, getStyles } from './StatusPills.styles';
+import { IStackTokens, Stack } from '@fluentui/react';
+import { useId } from '@fluentui/react-hooks';
+
+export const ColorPills: React.FC<IColorPillsProps> = (props) => {
+    const { visualColorings, width } = props;
+    const stackTokens: IStackTokens = { childrenGap: 2 };
+    const styles = getStyles(width);
+    const alignment = width === 'wide' ? 'center' : 'start';
+    const pillId = useId('cb-visual-rule-pills');
+    const [trimmedVisualColorings, setTrimmedVisualColorings] = useState(
+        visualColorings && visualColorings.length > 3
+            ? visualColorings.slice(0, 3)
+            : visualColorings
+    );
+    const [extraVisualColoringCount, setExtraVisualColoringCount] = useState(
+        visualColorings && visualColorings.length > 3
+            ? visualColorings.length - 3
+            : 0
+    );
+
+    useEffect(() => {
+        setTrimmedVisualColorings(
+            visualColorings && visualColorings.length > 3
+                ? visualColorings.slice(0, 3)
+                : visualColorings
+        );
+        setExtraVisualColoringCount(
+            visualColorings && visualColorings.length > 3
+                ? visualColorings.length - 3
+                : 0
+        );
+    }, [visualColorings]);
+
+    return (
```

Comment:
the ordering of the pills is not deterministic. I see Yellow, blue, green +1 on the elements panel and Blue, green, yellow (no plus 1???) on the modal.
![image](https://user-images.githubusercontent.com/57726991/201417482-064f929a-ad7e-4a19-b93d-2c404cb43b3b.png)
![image](https://user-images.githubusercontent.com/57726991/201417493-241290c5-361a-49fc-9c93-f1e896b66454.png)

Reply:
ahh. it's cause there is another behavior giving it another yellow. Interesting. Ok. So that's probably fine.